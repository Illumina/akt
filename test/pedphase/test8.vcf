chrX	5000120	.	ACCT	A	932	PASS	.	GT	1	1|1	0
chrX	5000293	.	GCA	G	339	PASS	.	GT	0	0|0	1
chrX	5000297	.	G	A	253	PASS	.	GT	0	0|0	1
chrX	5000394	.	G	A	264	PASS	.	GT	0	0|0	1
chrX	5000647	.	T	C	162	PASS	.	GT	0	0|1	0
chrX	5000774	.	C	A	319	PASS	.	GT	0	0|0	1
chrX	5000789	.	T	C	306	PASS	.	GT	0	0|0	1
chrX	5000857	.	A	G	325	PASS	.	GT	0	0|0	1
