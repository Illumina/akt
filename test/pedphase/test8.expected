#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	CephChild-12882	CephMother-12878	CephFather-12877
chrX	5000120	.	ACCT	A	932	PASS	.	GT:ME	1:0	1|1:.	0:.
chrX	5000293	.	GCA	G	339	PASS	.	GT:ME	0:0	0|0:.	1:.
chrX	5000297	.	G	A	253	PASS	.	GT:ME	0:0	0|0:.	1:.
chrX	5000394	.	G	A	264	PASS	.	GT:ME	0:0	0|0:.	1:.
chrX	5000647	.	T	C	162	PASS	.	GT:ME	0:0	0|1:.	0:.
chrX	5000774	.	C	A	319	PASS	.	GT:ME	0:0	0|0:.	1:.
chrX	5000789	.	T	C	306	PASS	.	GT:ME	0:0	0|0:.	1:.
chrX	5000857	.	A	G	325	PASS	.	GT:ME	0:0	0|0:.	1:.
